code/datums/religions.dm

```diff
@@ -0,0 +1,661 @@
+// This file lists all religions, as well as the prototype for a religion, and a proc to design your own if you want to.
+/datum/religion
+	// Following tradition, the default is Space Jesus (this is here to avoid people getting an empty relgion)
+	var/name = "Christianity"
+	var/deity_name = "Space Jesus"
+	var/bible_name = "The Holy Bible"
+	var/male_adept = "Chaplain"
+	var/female_adept = "Chaplain"
+	var/bible_type = /obj/item/weapon/storage/bible
+
+	var/obj/item/weapon/storage/bible/holy_book
+
+	var/list/bible_names = list()
+	var/list/deity_names = list()
+
+	var/list/keys = list() // What you need to type to get this particular relgion.
+
+/datum/religion/New() // For religions with several bibles/deities
+	if (bible_names.len)
+		bible_name = pick(bible_names)
+	if (deity_names.len)
+		deity_name = pick(deity_names)
+
+/datum/religion/proc/misc(var/mob/living/carbon/human/H, var/obj/item/weapon/storage/bible/B)
```

Comment:
Document what this method does with a comment and rename it to something like `equip_chaplain` too. What it does is completely non-obvious without looking at implementations.

Reply:
Did that, also the Bible thing was completely useless and I thought I had it removed several commits ago